README.md

```diff
@@ -0,0 +1,8 @@
+## Time Away App
+
+The application allows employees to register vacation dates and see colleagues vacations.
+
+### Run postgres
+
+To test application, use postgres docker container. To launch, you need to make sure that docker is installed on the
+local machine and use a simple command ```docker-compose up``` from the folder where the docker compose file is located.

```

Comment:
I would suggest using `docker-compose up -d`. And could you then also please describe how to stop it.